src/compas_view2/views/view.py

```diff
@@ -19,23 +19,20 @@ class View(QtWidgets.QOpenGLWidget):
         The view configuration.
     """
 
-    FRONT = 1
-    RIGHT = 2
-    TOP = 3
-    PERSPECTIVE = 4
+    VIEW_ANGLES = {"FRONT": 1, "RIGHT": 2, "TOP": 3, "PERSPECTIVE": 4}
 
     def __init__(self, app, view_config):
         super().__init__()
         self.setFocusPolicy(QtCore.Qt.StrongFocus)
         self._opacity = 1.0
-        self._current = View.PERSPECTIVE
+        self._current = self.VIEW_ANGLES[view_config["view_port"]]
         self.shader_model = None
         self.app = app
         self.color = view_config["background_color"]
         self.mode = view_config["view_mode"]
         self.selection_color = view_config["selection_color"]
         self.show_grid = view_config["show_grid"]
-        self.camera = Camera(self)
+        self.camera = Camera(self, **view_config["camera"])
```

Comment:
also to my point earlier, `view_config` could be interpreted as "viewing the config" (action + subject). it is clearer what is meant when `viewconfig` is used, in my opinion...